constants.py

```diff
@@ -12,3 +12,5 @@ class Constants:
     DRIVETRAIN_P = 0
     DRIVETRAIN_I = 0
     DRIVETRAIN_D = 0
+
+    DRIVETRAIN_RAMP = 100
```

Comment:
I'm going to fail this for now, first because that is not the desired value and second because I would like to test this on the actual robot.